addons/create-react-class/factory.js

```diff
@@ -243,6 +257,13 @@ function factory(ReactComponent, isValidElement, ReactNoopUpdateQueue) {
      */
     componentWillUpdate: 'DEFINE_MANY',
 
+    /**
+     * Replacement for (deprecated) `shouldComponentUpdate`.
+     *
+     * @optional
+     */
+    UNSAFE_shouldComponentUpdate: 'DEFINE_MANY',
```

Comment:
This is new to me. Why?

Reply:
Wow. Sorry! Looks like I fucked up there.

Reply:
Haha no worries. Fresh eyes always help 👀